docs/setup_toolchain.md

```diff
@@ -65,7 +65,7 @@ Next step: [Add keyboard support to Arduino](#add-keyboard-support-to-arduino)
     From a shell:
 
     ```sh
-    $ groups 
+    $ groups
     ```
 
     If you see dialout in the list, you're good to go. If you don't, you'll have to add yourself and then get Linux to recognize the change in your current shell:
```

Comment:
We're no longer dependent on `dialout`, so I think this can be ditched.

Reply:
Good call. Thanks. done.